frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/DeleteModal/DeleteModal.tsx

```diff
@@ -19,29 +21,39 @@ export type DeleteModalProps = {
  *
  * @property {boolean}[isOpen] - If the modal is open or not
  * @property {function}[onClose] - Function to execute on close
- * @property {function}[onDelete] - Function to execute on click delete
- * @property {string}[appName] - The name of the app to delete changes on
+ * @property {string}[app] - The app
+ * @property {string}[org] - The org
```

Comment:
I believe the term to describe this situation is "smør på flesk". We are utilizing TypeScript, and the properties are named 'app' and 'org,' which I find quite self-descriptive and then we do not need to add JSDoc for this? 🤔  It's just one more thing to maintain?